### DISCUSSION TITLE
Temporary forms of trust

### DISCUSSION THREAD

**Discussion by x1p** [GriefPrevention/GriefPrevention#703] [Ideas]
**As Player** I would suggest a new forms of trust that are similar to other game modes.
`add` : only trust a player when the claim owner is online. Similar to plotsquareds `plot add`
`coop` : only trust a player when the owner is online and will be retracted when the claim owner goes offline. Similar to Bskyblock `is team coop`
**Because** I think any form of temporary form of trust can be beneficial for players where they are a bit uncertain the trustee will behave when the owner is not in sight.

Many players give others trust, then the gameplay between them slowly becomes less and less, but they tent to forget to retract the trust. Mainly because they don't want to upset the other party by revoking the trust as it might feel offensive.
If they have the opportunity to give add, they can play together when both are online. And the change that the owner will be grieved during absence is lower.

Optional questions:
The challenge is how this can be integrated in the existing commands.
- as an option into the existing commands like `/trust add playername` or `/at add playername`
- or as a separate command like `/add playername`

**Is your feature request related to a problem? Please describe.**
Many players play together nicely but all to often a friendship won't last for ever and friends turn into grievers.

**Describe alternatives you've considered**
The only absurd alternative is plotsquared in a survival world with natural looking plots.
Furthermore staff can inform players as much as possible to prevent scams but there will always be players that look like friends and meanwhile steal. I'm looking for ways to prevent this by giving them an safe alternative.

**Comment by TomLewis** [comment]
This is a good idea.
